Makefile

```diff
@@ -3,15 +3,22 @@ export
 SHELL := /bin/bash
 LOCAL := $(PWD)/usr
 PATH := $(LOCAL)/bin:$(PATH)
-TESSDATA =  $(LOCAL)/share/tessdata 
-LANGDATA = $(PWD)/langdata-$(LANGDATA_VERSION)
+HOME := /home/ubuntu
+TESSDATA =  $(HOME)/tessdata_best
+LANGDATA = $(HOME)/langdata
```

Comment:
I find it preferable to have the explicit version of langdata for reproducible results.